docs/developer/other/RELEASE.md

```diff
@@ -90,13 +113,11 @@ $ docker push apereo/uportal-demo:latest
 ```
 
 ## Update Community
-For a Milestone or RC release, email to uportal-dev is needed.  Be sure to acknowledge those who contributed to the release.
-
-For a GA release also email uportal-user .  Be sure to put the release into context for an existing adopter to understand.
+For a patch release, email to `uportal-dev`.  Be sure to acknowledge those who contributed to the release.
```

Comment:
This whole section so be for any release, rather than type of release. Will update post-merge.